.devcontainer/hooks/shared/.env.example

```
@@ -0,0 +1,29 @@
+# Docker Compose Project Name
+# Automatically generated from git remote URL by initialize.sh
+# The initializeCommand in devcontainer.json runs this script before build
+# Must start with letter/number and contain only lowercase alphanumeric, hyphens, underscores
+COMPOSE_PROJECT_NAME=my-project
+
+# 1Password Service Account Configuration
+# Copy this file to .env and fill in your service account token
+# Get your service account token from: https://my.1password.com/developer-tools/infrastructure-secrets/
+
+OP_SERVICE_ACCOUNT_TOKEN="your-token-here"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)